### ISSUE TITLE
can't unlock solid blocks

### ISSUE THREAD

**Issue by meriweathern** [benbenlaw-modpacks/Techopolis-2#76]
I can create lapis and coal but no recipe seems to be available to use to create blocks ?
I can not also create any solid metal blocks copper tin etc.

I can not see it locked behind a quest ?

**Comment by Wishbone1977** [comment]
That seems... odd. I was able to create both lapis and coal blocks as soon as I got lapis and coal. Are you running the latest version of the pack, and have you modified it in any way?

**Comment by 4g0tt3nSou1** [comment]
I am having the same issue. Only modifications I have done is to swap Oculus, Rubidium for optifine, and updated ImmersiveEnginnering to compensate for optifine. Additionally, it seems the "Speed Caps" don't work either (manually "traded" ingots for blocks.)

**Comment by Wishbone1977** [comment]
This issue mentions the same thing happening after updating some of the mods: https://github.com/bbl-team/Techopolis-2/issues/67

Generally speaking, a given version of a modpack uses specific versions of the mods it contains, which are the versions it has been tested with. If you choose to update mods in a pack, all bets are off, essentially. There's no way of knowing if the changes to a newer version of a mod might break compatibility with something else other than testing it, and when you update mods yourself you invalidate the testing the pack maker has done.

In other words, this is not a bug in the pack, it's a bug you introduced by modifying the pack.

**Comment by 4g0tt3nSou1** [comment]
While I understand it introduces new variables, a little help figuring out the actual problem rather than writing it off would be nice(not trying to be rude :) ). I mean for me the pack shipped broken, wouldn't even launch because Oculus, Rubidium were causing mixin errors. I have not encountered a memory leak like #67, it more seems like somehow part of the game that handles "restricting" recipes, removed the recipes and never re-added them back on gamestage.

**Comment by Wishbone1977** [comment]
Well, I'm afraid I'm not in a position to be terribly helpful in that regard. I'm just a player like yourself. As you say, new variables have been introduced, and I have no way of knowing what they are. Even if I did it wouldn't help me much, because I also don't know what the existing variables are or how they influence one another.

I suppose you have already tried wiping the pack and installing it from scratch? Otherwise I don't really know what to suggest, but I do hope you find a solution to your problem.

**Comment by HumdrumPanic** [comment]
[4g0tt3nSou1](https://github.com/4g0tt3nSou1) The Discord: https://discord.gg/cSwGMqfu is a better place for the type of troubleshooting you are trying to figure out

[meriweathern](https://github.com/meriweathern) If you have modified the pack you'll want to close this issue, as Wishbone mentioned, it isn't within the scope of this github

**Comment by 4g0tt3nSou1** [comment]
Overall, I am curious where this is being caused. I did reach out in discord and nobody's said anything in reply. 

Seems like this surrounds the mod for restricting recipes and mods to gamestates. That being handled by KubeJS or something else?

**Comment by Wishbone1977** [comment]
Seems like the pack was just updated (twice), although curiously there have been no commits in this repository that I can see.

Still, it might have fixed the problems mentioned here, so maybe give it a go.

**Comment by meriweathern** [comment]
pure vanilla skyblocks single player install 
had to down grade to 2.0 to fix the problem.
